person.rb

```diff
@@ -2,13 +2,17 @@
 
 class Person
   attr_writer :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
+
+  has_many :rentals
+  has_many :books, through: :rentals
 
   def initialize(age, _name: 'Unknown', _parent_permission: true)
```

Comment:
- [ ] **[OPTIONAL]** Trying to create a Person object does not work and returns this error (see image). You could probably remove the preceding underscores so that you have something like this:

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
  ```

  ![ola-error](https://user-images.githubusercontent.com/36542986/151268893-9a49ba4e-8136-4626-a286-2a33a9c39f7a.jpg)